### PR TITLE
Create a database based on a diagram

### PR DESCRIPTION
**Pull request by GhostEsso**
Hello **Code Reviewer** :smiley:,
We hope you're doing well.
In this milestone, my partner @Nelofarzabi and I,
- took a look at the database diagram for a clinic;
- created a file named schema_based_on_diagram.sql where we implemented the database from the diagram;
- joined tables from many-to-many relationships;
- remembered to add the FK indexes;

Thank you for reviewing our project :pray:

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    type VARCHAR(50),
+    name VARCHAR(50)
+);
+
+CREATE TABLE invoices (
+    id SERIAL PRIMARY KEY,
+    total_amount DECIMAL , 
+    generated_at TIMESTAMP,
+    payed_at TIMESTAMP, 
+    medical_historY_id FOREIGN KEY
```
Comment (danmainah) [inline schema_based_on_diagram.sql]: - [ ] @Team great work adding all columns for the invoices table however the `medical_historY_id` needs to be in all lower case for accurate referencing `medical_history_id`

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+);
+
+CREATE TABLE invoice_items (
+    id SERIAL PRIMARY KEY,
+    unit_price DECIMAL,
+    quantity INTEGER,
+    total_price DECIMAL,
+    invoice_id FOREIGN KEY,
+    treatment_id FOREIGN KEY
+);
```
Comment (danmainah) [inline schema_based_on_diagram.sql:35]: - [ ] @GhostEsso and @Nelofarzabi Great work creating the vet clinic database, however, you forgot to add a join table" for the many-to-many relationship between `medical_histories` and `treatments`, you also forgot to add indexes on all FOREIGN KEYS.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @GhostEsso and @Nelofarzabi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️  Added all tables displayed in the database diagram.
✔️ Ensured all columns added are using the correct data type as shown in the database diagram.


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @GhostEsso, and @Nelofarzabi ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights! ✔️
- Created an all "join table" for the many-to-many relationships. ✔️
- Added indexes on all foreign keys✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@c1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by GhostEsso** [comment]
@danmainah @1GORDON  :pray:

**Pull request merged**